src/main/java/org/minions/devfund/reinaldo/battleship/Ocean.java

```diff
@@ -0,0 +1,161 @@
+package org.minions.devfund.reinaldo.battleship;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    private Ship[][] ships;
+    private int shotsFired; //int shotsFired - The total number of shots fired by the user.
+    private int hitCount; //int hitCount - The number of times a shot hit a ship.
+    private int shipsSunk; //The number of ships sunk. Remember that you have a total of 13 ships.
+
+    static final int SHIPLENGTH = 20;
+    private static final int SHIPS_QUANTITY = 13;
+
+    private static final List<Ship> SHIPS_TO_PLACE = Arrays.asList(
```

Comment:
Could you apply a Factory class here ?
Please take a look the code from another people ?
We reviewed this in the last meetings